src/components/Fieldset.tsx

```diff
@@ -0,0 +1,87 @@
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import React from "react";
+
+import "lbh-frontend/lbh/components/lbh-fieldset/_fieldset.scss";
+
+/**
+ * The proptypes for {@link FieldsetLegend}.
+ */
+export interface FieldsetLegendProps {
+  id?: string;
+  className?: string;
+  isPageHeading?: boolean;
+  children?: React.ReactNode;
+}
+
+/**
+ * The type of a valid fieldset legend component. It allows any React functional
+ * components that accept {@link FieldsetLegendProps} as proptypes.
+ */
+export type FieldsetLegendType = React.FunctionComponent<FieldsetLegendProps>;
+
+/**
+ * The legend for {@link Fieldset}.
+ */
+export const FieldsetLegend: FieldsetLegendType = (
```

Comment:
If we didn't export this, how would a user create one?

Reply:
Resolved in conversation, my comment didn't make sense.